### ISSUE TITLE
Place validation / error messages above input fields 

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/report-a-cybercrime#1458]
## Summary

To allow people to navigate to the missing input fields:

- Place privacy page validation above the checkbox.
- Place the what was affected validation on the what could be affected page above the checkbox list.

Consider adjusting copy accordingly for greater clarity.

For example: "Check the box **below**." / "Cochez la case **ci-dessous**."
